src/lib/Scenes/MyProfile/MyProfilePushNotifications.tsx

```diff
@@ -0,0 +1,315 @@
+// tslint:disable:no-empty
+import { Box, Button, color, Flex, Join, Sans, Separator } from "@artsy/palette"
+import { MyProfilePushNotificationsMutation } from "__generated__/MyProfilePushNotificationsMutation.graphql"
+import { updateMyUserProfileMutation } from "__generated__/updateMyUserProfileMutation.graphql"
+import { PageWithSimpleHeader } from "lib/Components/PageWithSimpleHeader"
+import colors from "lib/data/colors"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { renderWithPlaceholder } from "lib/utils/renderWithPlaceholder"
+import useAppState from "lib/utils/useAppState"
+import React, { useCallback, useState } from "react"
+import { Alert, Linking, RefreshControl, ScrollView, Switch, View } from "react-native"
+import { commitMutation, createRefetchContainer, graphql, QueryRenderer, RelayRefetchProp } from "react-relay"
+import { MyProfilePushNotifications_me } from "../../../__generated__/MyProfilePushNotifications_me.graphql"
+import { MyProfilePushNotificationsQuery } from "../../../__generated__/MyProfilePushNotificationsQuery.graphql"
+
+interface SwitchMenuProps {
+  onChange: (value: boolean) => void
+  value: boolean
+  title: string
+  description: string
+}
+
+export type PushNotificationPermissions =
+  | "receiveLotOpeningSoonNotification"
+  | "receiveNewSalesNotification"
+  | "receiveNewWorksNotification"
+  | "receiveOutbidNotification"
+  | "receivePromotionNotification"
+  | "receivePurchaseNotification"
+  | "receiveSaleOpeningClosingNotification"
+
+export const SwitchMenu = ({ onChange, value, title, description }: SwitchMenuProps) => (
+  <Flex flexDirection="row" alignItems="flex-start" flexShrink={0} my={1}>
+    <Flex style={{ width: "80%" }}>
+      <Sans size="4t" color="black100">
+        {title}
+      </Sans>
+      <Sans size="3t" color={colors["gray-semibold"]} py={0.5}>
```

Comment:
these are the old colors, this should be
```suggestion
      <Sans size="3t" color="black60" py={0.5}>
```

Reply:
oh thanks for letting me know